feature/certificates/install.sh

```diff
@@ -0,0 +1,11 @@
+. /feature-installer-utils.sh
+
+if [ "X$(cat /home/cloudcontrol/flavour)X" == "XazureX" ] || [ "X$(cat /home/cloudcontrol/flavour)X" == "XsimpleX" ] || [ "X$(cat /home/cloudcontrol/flavour)X" == "XtanzuX" ]
```

Comment:
How about
```
FLAVOUR="X$(cat /home/cloudcontrol/flavour)X"
if [  ${FLAVOUR} == "XazureX" ] || [ ${FLAVOUR} == "XsimpleX" ] || [ ${FLAVOUR} == "XtanzuX"  ]
```

This would save some execs.